src/tritonserver_stub.cc

```diff
@@ -694,10 +710,37 @@ TRITONBACKEND_ModelInstanceReportBatchStatistics()
 {
 }
 TRITONAPI_DECLSPEC void
+TRITONBACKEND_Initialize()
```

Comment:
Thanks for auditing... but these don't go in the stub. The stub is needed only for backend/server functions that are called from backend code itself. These functions are called from Triton into the backend.

Reply:
Ah okay. Fixed. Hopefully there was none in there that DID need go into the stub.